content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -1,22 +1,14 @@
 ---
-title: 'Get Started: Metrics, Parameters, and Plots'
-description: 'Get started with DVC parameters, metrics, and plots. Learn how to
+title: 'Get Started: Metrics, Plots, and Parameters'
+description: 'Get started with DVC metrics, plots, and parameters. Learn how to
```

Comment:
Not critical. The ordered just didn't make much sense to me anymore.

Reply:
I think it was just alphabetical